go/test/endtoend/onlineddl/vrepl_stress/onlineddl_vrepl_mini_stress_test.go

```diff
@@ -509,27 +512,30 @@ func runSingleConnection(ctx context.Context, t *testing.T, done *int64) {
 		case 2:
 			err = generateDelete(t, conn)
 		}
-		assert.Nil(t, err)
-		time.Sleep(singleConnectionSleepInterval)
+		select {
+		case <-ctx.Done():
+			log.Infof("Terminating single connection")
+			return
+		case <-time.After(singleConnectionSleepInterval):
+		}
+		if !errors.Is(err, context.DeadlineExceeded) { // this is an acceptable error
```

Comment:
Since we are handling `ctx.Done()` just above and returning, when will it come here with a `DeadlineExceeded`?

Reply:
You're absolutely right. Removing the `if` condition.